### ISSUE TITLE
Is the timeout correct?

### ISSUE THREAD

**Issue by sjh836** [patrickjuchli/basic-ftp#64]
* version : 3.4.2 , node 8.15
* code
```
const ftp = require("basic-ftp");

const client = new ftp.Client(300);
await client.access({
    host: ... ,
    port: ... ,
    user: ... ,
    password: ...
});
```
* result
```
Error: connect ETIMEDOUT xxx.xxx.xxx.xxx:21 (control socket)
at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1191:14)
```

It took about 21000 ms, not 300 ms.

**Comment by patrickjuchli** [comment]
This is a regression, thanks for the report.

**Comment by patrickjuchli** [comment]
Fixed in 3.4.3
